sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/SqliteCompiler.kt

```diff
@@ -15,28 +15,14 @@
  */
 package com.squareup.sqldelight
 
-import com.squareup.javapoet.ClassName
-import com.squareup.javapoet.FieldSpec
-import com.squareup.javapoet.JavaFile
-import com.squareup.javapoet.MethodSpec
-import com.squareup.javapoet.TypeSpec
-import com.squareup.sqldelight.model.body
-import com.squareup.sqldelight.model.constantName
-import com.squareup.sqldelight.model.identifier
-import com.squareup.sqldelight.model.isNullable
-import com.squareup.sqldelight.model.javaType
-import com.squareup.sqldelight.model.methodName
-import com.squareup.sqldelight.model.name
-import com.squareup.sqldelight.model.sqliteText
+import com.squareup.javapoet.*
```

Comment:
yes. star imports are the devil.

our code styles are at http://github.com/square/java-code-styles